virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -140,9 +140,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-33125: Add CPU limits to the UI
 * Cluster admins can now enable CPU resource limits on a namespace in the {product-title} web console under *Overview* -> *Settings* -> *Preview features*.
 
-[id="virt-4-16-bug-fixes"]
-== Bug fixes
-
+//CNV-22314: Safe memory overcommitment using `wasp-agent`
```

Comment:
Should this mention that it is in TP?

Reply:
@fabiand  This is in the 'Technology Preview features" section.